dockerfiles/python-3.13.Dockerfile

```
@@ -1,17 +1,19 @@
-COPY Pipfile /app/Pipfile
-COPY Pipfile.lock /app/Pipfile.lock
+# pipenv uses this to store virtualenvs
+ENV WORKON_HOME=/venvs
+
+RUN pip install --no-cache-dir "pipenv>=2025.0.2"
 
 WORKDIR /app
 
-ENV WORKON_HOME=/venvs
+# .git & README.md are unique per-repository. We ignore them on first copy to prevent cache misses
+COPY --exclude=.git --exclude=README.md . /app
```

Hadolint (2.12.0)

[error] invalid flag: --exclude

(DL1000)